lib/matplotlib/__init__.py

```diff
@@ -1417,19 +1417,15 @@ def tk_window_focus():
 
 
 def _init_tests():
-    try:
+    # CPython's faulthandler since v3.6 handles exceptions on Windows
```

Comment:
I'm not fully grokking this, but I thought we were supporting 3.5 for the next release...

Reply:
faulthandler exists since 3.3: https://docs.python.org/3.3/library/faulthandler.html

Reply:
Yes, but why does the comment make reference to "since v3.6"?

Reply:
It exists on all version (so we don't need the import check), but still need the version check to know if we want to use it on windows or not.